contracts/handlers/ERC20Handler.sol

```diff
@@ -68,6 +76,11 @@ contract ERC20Handler is IDepositHandler, ERC20Safe {
         // return true;
     }
 
+    function setBurnable(address contractAddress) public {
```

Comment:
In addition to the `setResourceIDAndContractAddress`, this is also a dangerous method to leave public without any other modifiers

Reply:
Would it be dangerous to add a method on the bridge that allows the validators to vote to execute an arbitrary call (like how multi-sigs work?) That way they could arbitrary call this method?

Reply:
what is arbitrary about calling this method? are you proposing we introduce another specialized voting mechanism for the burn list?